zipkin/src/main/java/com/linecorp/armeria/internal/tracing/SpanTags.java

```diff
@@ -73,6 +81,32 @@ public static void addTags(Span span, RequestLog log) {
         }
     }
 
+    private static String generateUrl(Scheme scheme, String authority, String path, @Nullable String query) {
+        final SessionProtocol sessionProtocol = scheme.sessionProtocol();
+        final String uriScheme;
+        if (SessionProtocol.httpValues().contains(sessionProtocol)) {
+            uriScheme = "http://";
+        } else if (SessionProtocol.httpsValues().contains(sessionProtocol)) {
+            uriScheme = "https://";
```

Comment:
is this one a pain to test? I noticed it wasn't in the test case, but not terribly fussed

Reply:
I can add one for sure.

Reply:
Added `SpanTagsTest` :smile: